modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderTableFDSView.java

```diff
@@ -41,7 +42,7 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 		).add(
 			"purchaseOrderNumber", "purchase-order-number"
 		).add(
-			"accountName", "account"
+			_addFDSTableAccountField()
```

Comment:
not sure Brian will like this method name
```suggestion
			_addAccountNameFDSTableSchemaField()
```

Reply:
In both classes

Reply:
✅